test/smoke_spec.js

```diff
@@ -0,0 +1,73 @@
+var PageObject = require('../page_objects/page_object_model')
+
+describe('smoke test demo', function() {
+var pageObject
+
+beforeEach(function(){
+
+  pageObject = new PageObject()
+
+  browser.waitForAngularEnabled(false);
+
+  browser.get('http://automationpractice.com/index.php');
+
+})
+
+
+      it('shall not smoke', function() {
```

Comment:
Test suite title and ID should match text inside "describe", and test case title and ID should match text inside "it".
Then, you will be in sync with documentation you wrote in docs.

For example, it should be 
"it('ST-001: Sign in with valid credentials, add item to shopping cart and successfully check out', function() { .."